main.py

```diff
@@ -529,6 +529,15 @@ def main():
                 args.interval_size = f['data'].shape[1]
                 args.batch_size = 1
 
+            # Ensure the sizes for intervals file and infer dataset are same.
+            infer_dataset = DatasetInfer(args.infer_files)
```

Comment:
This should only be run if `args.infer` is true, not `args.eval`.

Reply:
done